model/resource.py

```diff
@@ -759,10 +760,11 @@ def get(cls, _db, url, do_get=None, extra_request_headers=None,
             media_type = cls._best_media_type(url, headers, presumed_media_type)
             if isinstance(content, unicode):
                 content = content.encode("utf8")
-        except Exception, fetch_exception:
+        except Exception as e:
```

Comment:
Why not write `Exception as fetch_exception`?

Reply:
The variable name shadows any variable with the same name in the enclosing code block, and when the except block ends, `fetch_exception` actually goes out of scope. (I wasn't expecting this; I thought at most it would go back to its previous value).

Here's a simple illustration of the difference between Python 2 and Python 3 behavior:

```
error = None
print("Error starts out", error)
try:
    a = int("abc")
except Exception as error:
    print("I just caught", error)
print("Error is now", error)
```